app/views/layouts/application.html.erb

```diff
@@ -14,14 +14,13 @@
 		<script src="https://code.highcharts.com/highcharts.js"></script>
 		<script type="text/x-mathjax-config">
 			MathJax.Hub.Config({
-				extensions: ["tex2jax.js", "fast-preview.js"],
+				extensions: ["tex2jax.js", "Safe.js", "fast-preview.js"],
 				tex2jax: {
 					inlineMath: [['$','$'], ['\\(','\\)']],
 					processEscapes: true // Allow \$ for a literal $
 				},
-				jax: ["input/TeX", "output/HTML-CSS"],
-				messageStyle: "none",
-				showMathMenu: false,
+				showMathManu: true,
```

Comment:
I added those settings to keep it as non-intrusive as possible.
If you want them, I'm fine with that, but then you might want to change `showMathManu` to `showMathMenu`